docs/adrs/001-theming-peer-dependency.md

```diff
@@ -0,0 +1,39 @@
+# ADR-001: Theming peer dependency
+
+2023.04.24
+
+## Context
+
+Garden provides a collection of React components through multiple NPM packages.
+Each package has peerDependencies for `@zendeskgarden/react-theming`, `react`,
+`react-dom`, and `styled-components`. Typically, a package should increase its
+major version when there is a change in its peer dependency requirements.
+However, Garden proposes a DECISION to deviate from this [general
+rule](https://nodejs.org/en/blog/npm/peer-dependencies) and allow special
+treatment for `@zendeskgarden/react-theming` peer dependency.
```

Comment:
```suggestion
treatment for `@zendeskgarden/react-theming`.
```

The last part feels both extraneous and clunky to me as phrased. Alternatively, it could be `as a peer dependency`?